exercises/concept/blackjack/.docs/instructions.md

```diff
@@ -18,66 +18,45 @@ You will receive two cards and will be able to see the face up card of the deale
 
 Depending on your two cards and the card of the dealer, there is a strategy for the first turn of the game, in which you have the following options:
 
-    - Stand (S)
-    - Hit (H)
-    - Split (P)
-    - Automatically win (W)
+- Stand (S)
+- Hit (H)
+- Split (P)
+- Automatically win (W)
 
 Although not optimal yet, you will follow the strategy your friend Alex has been developing, which is as follows:
 
-Category: Large Hand
-
 - If you have a pair of aces you must always split them.
 - If you have a Blackjack (two cards that sum up to a value of 21), and the dealer does not have an ace, a figure or a ten then you automatically win. If the dealer does have any of those cards then you'll have to stand and wait for the reveal of the other card.
-
-Category: Small Hand
-
-- If your cards sum up to 17 or higher you should always stand.
-- If your cards sum up to 11 or lower you should always hit.
+- If your cards sum up to a value within the range [17, 20] you should always stand.
 - If your cards sum up to a value within the range [12, 16] you should always stand unless the dealer has a 7 or higher, in which case you should always hit.
-
-The overall logic has already been implemented. You have four tasks:
+- If your cards sum up to 11 or lower you should always hit.
 
 ## 1. Calculate the score of any given card.
 
-Implement a function to calculate the numerical value of a card given its name using conditionals.
+Implement a function to calculate the numerical value of a card:
 
 ```go
 value := ParseCard("ace")
 fmt.Println(value)
 // Output: 11
 ```
 
-## 2. Determine if two cards make up a Blackjack.
+## 2. Implement the strategy.
```

Comment:
the header here needs to match the header in the hints, otherwise the connection between task and hint might not work on the website

Reply:
Done.